content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -11,13 +11,13 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/linking-appl
 - [Working With Links and Labels](#working-with-links-and-labels)
 - [Next Steps](#next-steps)
 
-Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: key-value storage and NoOps SQL databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
+Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: key-value storage and SQLite Databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
```

Comment:
again, this was not previously captialised - next to "key-value storage" it looks weird to me but I don't know what the official branding guidelines say _grin_